dashboard/app/models/script_level.rb

```diff
@@ -276,6 +276,12 @@ def summarize(include_prev_next=true)
       ids.concat(l.contained_levels.map(&:id))
     end
 
+    # Levelbuilders can select if External/
+    # Markdown levels should display as Unplugged.
+    display_as_unplugged =
```

Comment:
Tiny nit: this is shadowing the `display_as_unplugged` definition added by the SerializedProperties concern.

https://github.com/code-dot-org/code-dot-org/blob/82960ff9e66bd4f932ea4f710f09e9d07bf058e4/dashboard/app/models/concerns/serialized_properties.rb#L58-L64